src/sections/Community/Handbook/writing-program.js

```diff
@@ -194,6 +194,7 @@ const Writers = () => {
             <ul>
               <li><a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-a-blog-post">Contributing to Layer5 Blogs</a></li>
               <li><a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-a-resource">Contributing to Layer5 Resources</a></li>
+              <li><a href="https://github.com/layer5io/layer5/labels/area%2Fblog">View open Layer5 Github Issues</a></li>
```

Comment:
```suggestion
              <li><a href="https://github.com/layer5io/layer5/labels/area%2Fblog">View open Writing Program issues</a></li>
```

I am incorporating @Nikhil-Ladha's feedback.